pkg/instancetype/instancetype_test.go

```diff
@@ -1373,12 +1373,99 @@ var _ = Describe("Instancetype and Preferences", func() {
 					},
 				}
 
-				conflicts := instancetypeMethods.ApplyToVmi(field, instancetypeSpec, preferenceSpec, &vmi.Spec)
+				conflicts := instancetypeMethods.ApplyToVmi(field, instancetypeSpec, preferenceSpec, &vmi.Spec, &vmi.ObjectMeta)
 				Expect(conflicts).To(HaveLen(1))
 				Expect(conflicts[0].String()).To(Equal("spec.template.spec.domain.devices.hostDevices"))
 			})
 		})
 
+		Context("Instancetype.Spec.Annotations", func() {
+
```

Comment:
nit: newline

Reply:
Most of the tests are formatted this way with a newline after the `Context` so I would stick with that.